src/components/projects/marc-medina/MarcMedina.tsx

```diff
@@ -1,39 +1,236 @@
+import {
+  GridProps,
+  LayoutProps,
+  PositionProps,
+  grid,
+  layout,
+  position,
+} from "styled-system";
+
 import Flex from "../../Flex";
-import Icon from "../Icon";
+import { Link } from "react-router-dom";
+import MarcMedinaPreview from "./MarcMedinaPreview";
+import { PROJECTS_URL } from "../../../constants/router-urls";
 import PreviewOrProjectPage from "../PreviewOrProjectPage";
 import React from "react";
-import Timer from "../../Timer";
+import image1 from "../../../assets/project-page/marc-medina/Mani_AntiVox_3.jpg";
+import image10 from "../../../assets/project-page/marc-medina/Mani_AntiVox_15.jpg";
+import image11 from "../../../assets/project-page/marc-medina/Mani_AntiVox_16.jpg";
+import image12 from "../../../assets/project-page/marc-medina/Mani_AntiVox_17.jpg";
+import image13 from "../../../assets/project-page/marc-medina/Mani_AntiVox_18.jpg";
+import image14 from "../../../assets/project-page/marc-medina/Mani_AntiVox_19.jpg";
+import image15 from "../../../assets/project-page/marc-medina/Mani_AntiVox_20.jpg";
+import image16 from "../../../assets/project-page/marc-medina/Mani_AntiVox_23.jpg";
+import image17 from "../../../assets/project-page/marc-medina/Mani_AntiVox_24.jpg";
+import image18 from "../../../assets/project-page/marc-medina/Mani_AntiVox_25.jpg";
+import image19 from "../../../assets/project-page/marc-medina/Mani_AntiVox_26.jpg";
+import image2 from "../../../assets/project-page/marc-medina/Mani_AntiVox_4.jpg";
+import image20 from "../../../assets/project-page/marc-medina/Mani_AntiVox_27.jpg";
+import image21 from "../../../assets/project-page/marc-medina/Mani_AntiVox_28.jpg";
+import image22 from "../../../assets/project-page/marc-medina/Mani_AntiVox_29.jpg";
+import image3 from "../../../assets/project-page/marc-medina/Mani_AntiVox_5.jpg";
+import image4 from "../../../assets/project-page/marc-medina/Mani_AntiVox_6.jpg";
+import image5 from "../../../assets/project-page/marc-medina/Mani_AntiVox_8.jpg";
+import image6 from "../../../assets/project-page/marc-medina/Mani_AntiVox_9.jpg";
+import image7 from "../../../assets/project-page/marc-medina/Mani_AntiVox_10.jpg";
+import image8 from "../../../assets/project-page/marc-medina/Mani_AntiVox_11.jpg";
+import image9 from "../../../assets/project-page/marc-medina/Mani_AntiVox_14.jpg";
 import mask from "../../../assets/project-page/project-icons/mask.png";
+import styled from "styled-components";
+import theme from "../../theme";
+
+const Images: ImageProps[] = [
```

Comment:
this isnt a React component so should just be `const images = ` lowercase

Reply:
Good spot Justen, good luck on the new contract!